.github/workflows/flake8.yml

```
@@ -26,4 +26,4 @@ jobs:
         pip install flake8==4.0.1
     - name: Lint with flake8
       run: |
-        flake8 .
\ No newline at end of file
+        flake8 *.py && flake8 listeners/
```

I think we can combine the two commands into one

```suggestion
        flake8 *.py listeners/
```

---

I was going to do so! but I works a bit differently